src/ApiCheck/Baseline/MemberBaseline.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace ApiCheck.Baseline
+{
+    public class MemberBaseline : BaselineItem
+    {
+        public override string Id
+        {
+            get
+            {
+                return string.Join(" ", GetComponents());
+            }
+        }
+
+        private IEnumerable<string> GetComponents()
+        {
+            if (ExplicitInterface == null)
+            {
+                switch (Visibility)
+                {
+                    case BaselineVisibility.Public:
+                        yield return "public";
+                        break;
+                    case BaselineVisibility.Protected:
+                        yield return "protected";
+                        break;
+                    case BaselineVisibility.Internal:
+                        yield return "internal";
+                        break;
+                    case BaselineVisibility.ProtectedInternal:
+                        yield return "protected";
+                        yield return "internal";
+                        break;
+                    case BaselineVisibility.Private:
+                        yield return "private";
+                        break;
+                    default:
+                        break;
+                }
+            }
+
+            if (Constant)
+            {
+                yield return "const";
+            }
+            else
+            {
+                if (Static)
+                {
+                    yield return "static";
+                }
+
+                if (ReadOnly)
+                {
+                    yield return "readonly";
+                }
+            }
+
+            if (Abstract)
+            {
+                yield return "abstract";
+            }
+
+            if (Sealed && ImplementedInterface == null)
+            {
+                yield return "sealed";
+            }
+
+            if (!Sealed && Virtual && !Abstract && !Override && ImplementedInterface == null)
+            {
+                yield return "virtual";
+            }
+
+            if (Override)
+            {
+                yield return "override";
+            }
+
+            if (New)
+            {
+                yield return "new";
+            }
+
+            if (ReturnType != null)
+            {
+                yield return ReturnType;
+            }
```

Comment:
The return type is System.Void for void returning methods. ReturnType == null for constructors only